### DISCUSSION TITLE
"InputMerge" and "ChannelSplitOutput" classes examples?

### DISCUSSION THREAD

**Discussion by willdatrill2007** [pschatzmann/arduino-audio-tools#1077] [Q&A] [answered]
Any examples using the "InputMerge" and the "ChannelSplitOutput" classes?

**Comment by pschatzmann** [comment, marked as answer]
I managed to [add a merge example today](https://github.com/pschatzmann/arduino-audio-tools/blob/main/examples/examples-stream/streams-generator-merge-pwm/streams-generator-merge-pwm.ino)

**Reply by willdatrill2007** [reply]
Hey, that's great. Thanks.

**Reply by noisetaub** [reply]
Thanks, what about ChannelSplitOutput? it's possible to split a sd audioplayer

**Reply by pschatzmann** [reply]
Sure, you can use any class that can act as audio sink!
